functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/newtests/HistoricalChangesNewTestsSelectionExecutionFunctionalTest.java

```diff
@@ -47,7 +51,9 @@ public void should_only_execute_new_test_when_multiple_commits_with_changed_test
 
         // when
         final List<TestResult> actualTestResults = project
-            .withEnvVariables("git.last.commits", "3")
+            .buildOptions()
+                .withSystemProperties("git.last.commits", "3")
+                .configure()
```

Comment:
Can we have this discussion in one place? 
https://github.com/arquillian/smart-testing/pull/51#discussion_r126934151